pcsx2/DebugTools/SymbolMap.h

```diff
@@ -19,6 +19,7 @@
 #include <set>
 #include <map>
 #include <string>
+#include <mutex>
 
 #include "Utilities/Threading.h"
```

Comment:
`pcsx2\debugtools\SymbolMap.h(148): error C2039: 'recursive_mutex': is not a member of 'std'`
Errors out if I remove the include.